code/game/objects/effects/holy.dm

```diff
@@ -0,0 +1,36 @@
+//Effects used for anything holy and divine currently used for smite and spawming a admin dummy... maybe for holy explosion one day
+/obj/effect/holy
+	name = "holy"
+	icon = null
+	icon_state = null
+	pixel_y = 0
+	pixel_x = 0
+	mouse_opacity = 0
+
+	proc/start(atom/location)
+		loc = get_turf(location)
+		spawn(20)
+			qdel(src)
+
+/obj/effect/holy/lightning
+	name = "divine retribution"
+	icon = 'icons/effects/224x224.dmi'
```

Comment:
Are you sure about that? This icon file already existed I did not make a new file for this I just exported the icon state from the pre rebase code and imported it back into the file with the same name and a did a few edits as it had null frames.
So do you want me to leave it or make a new file something like holy.dmi with only the lightning and holy light effect icons and put it in the icons/hippie?